octo/package-buildpack.sh

```diff
@@ -8,6 +8,8 @@ if [[ -n "${PUBLISH+x}" ]]; then
     --config "${HOME}"/package.toml \
     --publish
 
+  crane tag "${PACKAGE}:${VERSION}" "${VERSION_MINOR}"
+  crane tag "${PACKAGE}:${VERSION}" "${VERSION_MAJOR}"
```

Comment:
Because we're in an if block that is checking for PUBLISH to be set, I think that we should always have VERSION_MINOR/MAJOR set (based on other logic in the pipelines), but I would prefer to be defensive here. Can you make it check and only tag if the VERSION_MINOR/MAJOR is non-empty?